package.json

```diff
@@ -82,6 +82,8 @@
     "react-notification": "^6.8.5",
     "react-overlays": "5.2.0",
     "react-phone-number-input": "^3.2.11",
+    "react-qr-reader": "^3.0.0-beta-1",
```

Comment:
Nice, I love beta™️

Reply:
I also despise the fact that we need two whole deps for this

Reply:
Tbf, this is the official released latest version (dunno why they branded it as beta), and this package is the most popular qr-scanner for React out there. Also, these two packages combined only amounts to less than 100kb increase of the bundle size. So this should not be a concern.

Reply:
100Kb is not really a _small_ increase in bundle size. It will add about 5% to the total bundle size of `lego-webapp`, but as it should only be loaded on the user-validator page it's not really a problem.

Reply:
Yeah, that is a good point. But as you said, since this increase only is in the validator, it is acceptable. QR code generator on the user profile page adds about 4kb for reference.